django/db/models/fields/related.py

```diff
@@ -923,6 +923,21 @@ def validate(self, value, model_instance):
                 },  # 'pk' is included for backwards compatibility
             )
 
+    def resolve_related_fields(self):
+        related_fields = super().resolve_related_fields()
+        for _from_field, to_field in related_fields:
```

Comment:
@felixxm , I have replaced `_` in your suggested patch with `_from_filed` because of the `flake8` check error:
`F402 import '_' from line 17 shadowed by loop variable`.

Let me know If you have another suggestion about it.